packages/fx-core/src/plugins/resource/bot/aadRegistration.ts

```diff
@@ -25,29 +27,34 @@ export class AADRegistration {
 
     const result = new BotAuthCredential();
 
-    // 1. Register a new AAD App.
-    let regResponse = undefined;
-    try {
-      regResponse = await axiosInstance.post(
-        `${AADRegistrationConstants.GRAPH_REST_BASE_URL}/applications`,
-        {
-          displayName: displayName,
-          signInAudience: AADRegistrationConstants.AZURE_AD_MULTIPLE_ORGS,
-        }
-      );
-    } catch (e) {
-      throw new ProvisionError(CommonStrings.AAD_APP, e);
-    }
-
-    if (!regResponse || !utils.isHttpCodeOkOrCreated(regResponse.status)) {
-      throw new ProvisionError(CommonStrings.AAD_APP);
+    if (!objectId && !msAppId) {
```

Comment:
What if only objectId assigned or only msAppId assigned?
```
objectId = "xxx-xxxx";
msAppId = undefined;
```

Reply:
Exactly. I mean we don't register new aad when one of the ids is undefined. Does the bot works in that case?